common/script/api-v3/errors.js

```diff
@@ -0,0 +1,9 @@
+import CustomError from './customError';
+
+export class NotAuthorized extends CustomError {
```

Comment:
I think it's fine to use the http code errors. It's the website that's using the shared code, so we can just ignore the http code part of it.